pallets/xtransfer/src/xcm/xcm_transfer.rs

```diff
@@ -161,8 +161,9 @@ pub mod pallet {
 				fee: (MultiLocation::new(1, Here), Fungible(0u128)).into(),
 				origin: MultiLocation::new(1, X1(Parachain(1u32))),
 				dest_location: MultiLocation::new(1, X1(Parachain(2u32))),
-				recipient: pallet_assets_wrapper::ASSETS_REGISTRY_ID.into_account(),
+				beneficiary: GeneralKey(b"recipient".to_vec()).into(),
```

Comment:
Could it be too small? I suggest to make it at least 32 bytes long

Reply:
This is a dummy function used to estimate weight

Reply:
But if it's not enough long, can it underestimate the weight?

Reply:
Maybe have a little bit difference, anyway, let's use AccountId32 instead.